### ISSUE TITLE
Tall Grass

### ISSUE THREAD

**Issue by MopeyBoop** [warpKaiba/TMpokeheroes#4]
Hey, it's me again, bugging you.

Is there any possibility to have an assist for the Tall Grass section? The egg position is fixed, so it might be possible to ascertain where the egg is positioned every time. Though, I guess this could also be serverside? Or a bit, uh, very apparent with how fast it would be going, especially for the Legendary hunts?

**Comment by warpKaiba** [comment]
The position is server side, but it's possible to deduce possibilities of where it is based on where you've checked and what the response was. (Close, far away etc) When my new account gets to the point where I can access all the features I might start fixing everything to work again which is the highest priority rn

**Comment by MopeyBoop** [comment]
At the risk of sounding obnoxiously ignorant... does the response from the server only give the vague position marker, but not the position itself? I imagine it would have to compare clicked location and output into the Tall Grass, but I suppose if that could be read, you'd already have found out about it. ^^;

**Comment by warpKaiba** [comment]
Yeah it only sends back the text that displays that says how far away it is
